config/settings.yml

```diff
@@ -55,3 +55,9 @@
      facebook_app_id: 'overwrite in settings.local.yml or in an environment settings file'
      share_progress_api_key: 'Get this from your ShareProgress instance and include in a .local file or environment setting file.'
 
+# Braintree connection information. Used for taking donations.
+     braintree:
```

Comment:
If something goes into production settings files, it should have a corresponding entry in `settings.yml`, for use as a quick reference.